src/js/disability-benefits/utils/appeal-helpers.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export const appealStatusDescriptions = (eventType) => {
```

Comment:
Do react nodes behave properly if this is a javascript object?  I made this a functional component which seemed to accomplish what I needed